### PR TITLE
Update Dockerfile with GD extension

### PR DESCRIPTION
**Pull request by Gomusss**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -52,10 +52,12 @@ RUN set -eux ; \
     intl \
     pcntl  \
     curl \
+    gd \
     exif ; \
     pecl install amqp ; \
     pecl install xdebug; \
     pecl install imagick ; \
```
Comment (Daniel-Marynicz) [inline Dockerfile:71]: required to execute   
```
 docker-php-ext-configure gd   \
--with-webp \
--with-jpeg \
--with-xpm \
--with-freetype 

```
i you want add support for this files  types.
 and you need with apk install packages 


libpng-dev 
        jpeg-dev  
        freetype-dev  
        libwebp-dev 
        libzip-dev 


and

        libpng 
        libjpeg  
        freetype
        libwebp

**Review by Daniel-Marynicz — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -45,17 +48,28 @@ RUN set -eux ; \
...
     pdo_pgsql \
     intl \
     pcntl  \
     curl \
+    gd \
     exif ; \
     pecl install amqp ; \
     pecl install xdebug; \
     pecl install imagick ; \
+    docker-php-ext-configure gd   \
```
Comment (Daniel-Marynicz) [inline Dockerfile]: this must be before docker-php-ext-install

File: Dockerfile [hunk 3]
```diff
@@ -45,17 +48,28 @@ RUN set -eux ; \
         file \
         curl-dev \
         libpng-dev \
+        jpeg-dev \ 
+        freetype-dev \
+	libwebp-dev \
```
Comment (Daniel-Marynicz) [inline Dockerfile]: please format this line like other lines.

**Review by Daniel-Marynicz — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (Gomusss) [reply Dockerfile]: Moved before docker-php-ext-install

File: Dockerfile (hunk 3 above)
Comment (Gomusss) [reply Dockerfile]: Already formatted

**Review by Daniel-Marynicz — APPROVED** [review]

**Review by Daniel-Marynicz — APPROVED** [review]

**Pull request merged**